### ISSUE TITLE
Fails to compile on npm install

### ISSUE THREAD

**Issue by catid** [murat-dogan/node-datachannel#35]
Errors:

```
npm ERR! info RUN cmake "/root/git/boss-balloon/node_modules/node-datachannel" --no-warn-unused-cli -G"Unix Makefiles" -DCMAKE_JS_VERSION="6.1.0" -DCMAKE_BUILD_TYPE="Release" -DCMAKE_LIBRARY_OUTPUT_DIRECTORY="/root/git/boss-balloon/node_modules/node-datachannel/build/Release" -DCMAKE_JS_INC="/root/.cmake-js/node-x64/v16.0.0/include/node;/root/git/boss-balloon/node_modules/node-datachannel/node_modules/n
an" -DCMAKE_JS_SRC="" -DNODE_RUNTIME="node" -DNODE_RUNTIMEVERSION="16.0.0" -DNODE_ARCH="x64"
npm ERR! info CMD BUILD                 
npm ERR! info RUN cmake --build "/root/git/boss-balloon/node_modules/node-datachannel/build" --config Release                                                                  
npm ERR! /root/git/boss-balloon/node_modules/node-datachannel/src/peer-connection-wrapper.cpp: In lambda function:
npm ERR! /root/git/boss-balloon/node_modules/node-datachannel/src/peer-connection-wrapper.cpp:465:36: error: aggregate ‘std::ostringstream stream’ has incomplete type and cannot be defined
npm ERR!   465 |                 std::ostringstream stream;  
```

**Comment by catid** [comment]
Needs #include <sstream> in this file

**Comment by murat-dogan** [comment]
Which version are you using?

I can see include at latest branch
https://github.com/murat-dogan/node-datachannel/blob/bfb842512a040f978a9cac3061f93e97375447be/src/peer-connection-wrapper.cpp#L3

**Comment by catid** [comment]
node-datachannel@0.1.2

**Comment by catid** [comment]
"node-datachannel": "^0.1.2",

**Comment by catid** [comment]
This downloads the tgz: `https://registry.npmjs.org/node-datachannel/-/node-datachannel-0.1.2.tgz`

This file is missing the change you quoted

**Comment by catid** [comment]
Also missing in the latest release:
https://github.com/murat-dogan/node-datachannel/releases/tag/v0.1.2

**Comment by murat-dogan** [comment]
Thanks for reporting.
You are right, I corrected this only on master branch.
I released v0.1.3.
